lib/parsemrc.js

```diff
@@ -16,8 +16,6 @@ module.exports = {
   useRestrictionGroupDesignationCheck: {},
   useRestrictionGroupDesignationReport: {},
 
-  nonNumericBarcodesOkay: false,
```

Comment:
Looks like this determines whether or not it's safe to run parseInt on barcodes to improve lookup speeds (because Array.indexOf is faster if the elements are all ints, I think?). Looks like the code now calls parseInt regardless (so we are utilizing that speedup, which is good), but it seems like this could break if we encounter any barcodes that are not numeric - but I'm not sure if that ever happens. Have you run across any barcodes that are non-numeric?

Reply:
Yeah, I was going back and forth on this. I believe it's handled well enough to not need this part of the code. Still... might be safer to put it back in. There's a bunch of code that applies to this non-numeric barcode handling. 

I did run across several non-numeric barcodes using the scsb-export code, and it ran fine, as far as I could tell, but ... if you want me to put it back JIC, I will.

Reply:
Real question I was wondering: under what circumstance would it be ok to process non-numeric barcodes?

Reply:
True. Given that non-numeric barcodes are never legit anyway - and we're presently logging them when we find them - it's fine to assume they're numeric.